configure.py

```diff
@@ -1879,6 +1892,8 @@ def configure_abseil(build_dir, mode, mode_config):
                     for src in srcs
                     if src.endswith('.cc')]
             objs.append('$builddir/../utils/arch/powerpc/crc32-vpmsum/crc32.S')
+            if has_wasmtime:
+                objs.append('/usr/lib64/libwasmtime.a')
```

Comment:
I'm not exactly sure why, but when using `-lwasmtime` I was getting duplicate symbol erros when linking. I didn't look that much into it because this wasmtime dependency is going to be removed anyway

Reply:
Actually, this was when `-lwasmtime` was added to the `libs` list, perhaps here in `objs` this could still work - I'll check

Reply:
so, did you check it? Any conclusions?

Reply:
I did check it, but unfortunately it still doesn't work (if I put it in `objs` ninja wants a rule to make `-lwasmtime`)